mora/app.py

```diff
@@ -8,14 +8,16 @@
 
 import os
 
-import werkzeug
 import flask
+import werkzeug
+from flask_session import Session
 
-from . import auth
 from . import cli
 from . import exceptions
 from . import service
+from . import settings
 from . import util
+from .auth import auth, sso
```

Comment:
`auth.auth` feels kind of… iffy. Could we perhaps name the module something that's ever so slightly less repetitive?